test/e2e/framework/pv_util.go

```diff
@@ -233,15 +290,18 @@ func CreatePVCPV(c clientset.Interface, pvConfig PersistentVolumeConfig, pvcConf
 
 	By(fmt.Sprintf("Creating a PVC followed by a%s PV", preBindMsg))
 	// instantiate the pvc
-	pvc = CreatePVC(c, ns, pvc)
+	if pvc, err = CreatePVC(c, ns, pvc); err != nil {
```

Comment:
@sttts Is the issue you have multiple assignments preceded by `if`, or using the `if <func call>; <condition>` any time?
There are ~279 case where `if ... err := func-call(); err <operation>` in the e2e tests, so it is not an anti pattern in my opinion.